flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParserCalcitePlanner.java

```diff
@@ -1215,7 +1217,8 @@ private RelNode genGBRelNode(
             gbInputRexNodes.add(cluster.getRexBuilder().makeInputRef(srcRel, 0));
         }
 
-        return LogicalAggregate.create(gbInputRel, groupSet, transformedGroupSets, aggregateCalls);
+        return LogicalAggregate.create(
+                gbInputRel, ImmutableList.of(), groupSet, transformedGroupSets, aggregateCalls);
```

Comment:
Current call of `LogicalAggregate#create` https://github.com/apache/calcite/blob/cd5229b9a371e20de207231d55bcbddf1ff39ec2/core/src/main/java/org/apache/calcite/rel/logical/LogicalAggregate.java#L120-L124 is identical to https://github.com/apache/calcite/blob/cd5229b9a371e20de207231d55bcbddf1ff39ec2/core/src/main/java/org/apache/calcite/rel/logical/LogicalAggregate.java#L111-L116 with empty list of hints